CMakeLists.txt

```diff
@@ -355,6 +355,8 @@ if(BUILD_TORCH)
       CMAKE_COMMAND=${CMAKE_CMD}
       CUDA_TOOLKIT_ROOT_DIR=${CUDA_TOOLKIT_ROOT_DIR}
       USE_CUDA=${USE_CUDA}
+      CMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
```

Comment:
`CMAKE_<LANG>_COMPILER` indicates the full path of compiler, but you're using them to pass flags? https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER.html

Reply:
Hi, do you recommend that we use different names to pass the flags ? such as `MAIN_CMAKE_{LANG}_COMPILER`?